components/calculator/Step1.vue

```diff
@@ -95,10 +98,17 @@ const props = defineProps<{
 }>();
 
 const typeOfRoofWithCoeff = [
-  { label: "Ardoise", name: "ardoise", value: 0.8 }, // "slate"
-  { label: "Tuile / tôle", name: "tuile", value: 0.9 }, // "tile"
-  { label: "Plat", name: "plat", value: 0.6 }, // "flat"
-  { label: "Végétalisé", name: "vegetal", value: 0.4 }, // "planted"
+  { label: t("step1.slate"), name: "ardoise", value: 0.8 },
+  { label: t("step1.tile"), name: "tuile", value: 0.9 },
+  { label: t("step1.flat"), name: "plat", value: 0.6 },
+  { label: t("step1.vegetal"), name: "vegetal", value: 0.4 },
 ];
 
+// workaround to pass the titles, may be ugly
+const stepTitles = {
+  substep1: t("step1.roof_type_prompt"),
+  substep2: t("step1.draw_roof_prompt"),
+  substep3: t("step1.mains_drainage_prompt")
+};
```

Comment:
haaan i see what you did there!

This may be useless, see my comments above (or after, depending where your read this one ^^)